docs/javascript/v2/advanced-features/audio-level.mdx

```diff
@@ -7,31 +7,78 @@ nav: 5.01
 
```

Comment:
why is this change here. don't create branches from what you have. Try to create a new from the latest main.